packages/govuk-frontend/src/govuk/components/table/_index.scss

```diff
@@ -43,23 +43,21 @@
   }
 
   // Modifiers that make captions look more like their equivalent headings
+  .govuk-table__caption--xl,
+  .govuk-table__caption--l,
+  .govuk-table__caption--m {
+    margin-bottom: govuk-spacing(3);
+  }
```

Comment:
Thanks for combining all these @owenatgov

Out of interest, how come we don't use the heading classes?

We should fix this example in the YAML where we've used one by accident:

https://github.com/alphagov/govuk-frontend/blob/16d0085e072c995df5107a6055a1328a57631b99/packages/govuk-frontend/src/govuk/components/table/table.yaml#L135

Reply:
I'll fix that example but the question itself feels like one for https://github.com/alphagov/govuk-frontend/issues/4277

Reply:
Brill, thank you

Reply:
I just spotted the following in our docs for `captionClass` on tables:

>Classes for caption text size. Classes should correspond to the available typography heading classes.

Oh dear. More mounting evidence for https://github.com/alphagov/govuk-frontend/issues/4277 I'm not gonna let that stop me removing the heading class but I'm instead going to replace it with one of the caption modifiers for consistency _for now_.

Reply:
Ahh so the YAML docs said ["typography heading classes" since 2018](https://github.com/alphagov/govuk-frontend/pull/998) but must have forgotten to update that line when [caption classes came in 2021](https://github.com/alphagov/govuk-design-system/pull/1478). Will leave it with you

Reply:
> Out of interest, how come we don't use the heading classes?

We've generally tried to avoid calling things headings if they aren't semantically headings.

Reply:
Thanks @36degrees

Squeeze in a `captionClass` YAML docs fix if you like @owenatgov?